Parser/lexer/lexer.c

```diff
@@ -59,6 +59,10 @@ tok_nextc(struct tok_state *tok)
     int rc;
     for (;;) {
         if (tok->cur != tok->inp) {
+            if (INT_MAX - tok->col_offset - 1 < 0) {
```

Comment:
Why not simply `tok->col_offset >= INT_MAX`?

Reply:
If `tok->col_offset` is non-negative, `INT_MAX - tok->col_offset - 1 < 0` is the same as `tok->col_offset >= INT_MAX`. If `tok->col_offset` is negative, `INT_MAX - tok->col_offset` is an undefined behavior.

Maybe write it as `(unsigned int)tok->col_offset >= (unsigned int)INT_MAX`?